app/frontend/app/templates/notifications.hbs

```diff
@@ -1,11 +1,35 @@
 <div class="container">
   <div class="row" style="margin-top: 50px">
-    <div class="col-md-6 col-md-offset-3">
+    <div class="col-md-10 col-md-offset-1">
       <ul class="all-notifications">
-        {{#each}}
-          {{partial "notification"}}
+        {{#each notification in model}}
+          <div {{bind-attr class=":notification-item notification.seen::new"}}>
+            <div class="notification-title">
+              <a {{bind-attr href=notification.link}}>
+                <div class="avatar">
+                  {{avatar notification.sourceUser 'thumb_small'}}
```

Comment:
Yes, as it currently is, all notifications are forced to have a source. We might need to change that if we want to bundle notifications. Unless we do something like _<source> and 3 others..._ to prevent bigger schema changes.